test/util/timeout-registrySpec.ts

```diff
@@ -121,7 +126,10 @@ describe('timeout registry', () => {
 
         it('clears timeout when connection lost', done => {
```

Comment:
Does this test do the same thing that the test above? They have the same description.